src/Jackett.Common/Definitions/arabp2p.yml

```diff
@@ -110,20 +109,29 @@ login:
 
 search:
   paths:
-    # https://www.arabp2p.net/index.php?page=torrents&search=&category=0&active=0&internel=0&order=3&by=2
     - path: index.php
   inputs:
     page: torrents
     search: "{{ .Keywords }}"
-    # does not support multi categories, default to All
```

Comment:
Please restore all of the original comments you have deleted, I find them useful when maintaining indexers.
Thanks.